cav_srvs/CMakeLists.txt

```diff
@@ -14,211 +14,49 @@
 # the License.
 #
 
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(cav_srvs)
-
-## Compile as C++14, supported in ROS Noetic and newer
-# add_compile_options(-std=c++14)
-
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  message_generation
-  std_msgs
-  cav_msgs
-  geometry_msgs
+set(CMAKE_CXX_STANDARD 1 4)
+
+find_package(rosidl_default_generators REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(cav_msgs REQUIRED)
+# find_package(geometry_msgs REQUIRED)
+
+rosidl_generate_interfaces(
+    ${PROJECT_NAME}
+    srv/AbortActiveRoute.srv
```

Comment:
Is it possible to still use the glob logic from the hold packages here?
```
file(GLOB srv_files srv/*.srv)
set(srvs )
foreach(filePath ${srv_files})
  get_filename_component(name ${filePath} NAME)
  set(srvs ${srvs} ${name})
endforeach(filePath)
```
then is 
```
rosidl_generate_interfaces(
    ${PROJECT_NAME}
    ${srvs}
```

While not as explicit for the message only packages it is more convenient to be able to just drop the additional new messag in and have it compile. 

Same idea applies to the .msg files in cav_msgs

Reply:
I can look into this now.